tt_eager/tt_dnn/op_library/transformer_tms/multi_core_ssm_prefix_scan/multi_core_ssm_prefix_scan.cpp

```diff
@@ -88,7 +88,12 @@ operation::ProgramWithCallbacks multi_core_ssm_prefix_scan(
     const uint32_t cb_zeros_id = tt::CB::c_intermed6;
     const auto cb_zeros = create_circular_buffer(cb_zeros_id, 1, intermediary_tile_size, intermediary_format);
 
-    std::vector<uint32_t> reader_compile_time_args = {cb_a_in_id, cb_bx_in_id};
+    const uint32_t cb_h_acc_id = tt::CB::c_intermed7;
+    const uint32_t num_chunks_per_row = ceil(float(total_tiles_per_row) / 32.0f);
```

Comment:
`div_up` might be cleaner for you without having to do these casts. Also use some variable for `32` const.